cli/replicasets/list.go

```diff
@@ -0,0 +1,138 @@
+package replicasets
+
+import (
+	"fmt"
+	"sort"
+	"strconv"
+	"strings"
+
+	"github.com/apex/log"
+	"github.com/tarantool/cartridge-cli/cli/common"
+	"github.com/tarantool/cartridge-cli/cli/context"
+)
+
+const (
+	instanceMarker       = "•"
+	leaderInstanceMarker = "★"
+)
+
+func List(ctx *context.Ctx, args []string) error {
+	var err error
+
+	if err := FillCtx(ctx); err != nil {
+		return err
+	}
+
+	conn, err := connectToSomeJoinedInstance(ctx)
+	if err != nil {
+		return err
+	}
+
+	topologyReplicasets, err := getTopologyReplicasets(conn)
+	if err != nil {
+		return fmt.Errorf("Failed to get current topology replica sets: %s", err)
+	}
+
+	replicasetsSummary := getTopologyReplicasetsSummary(topologyReplicasets)
+
+	log.Infof("Current replica sets:\n%s", replicasetsSummary)
+
+	return nil
+}
+
+func getTopologyReplicasetsSummary(topologyReplicasets *TopologyReplicasets) string {
+	replicasetsSummaryByAliases := make(map[string]string)
+	replicasetsAliases := make([]string, len(*topologyReplicasets))
+
+	// sort replicasets summaries by aliases
+
+	i := 0
```

Comment:
This block looks big:
- Get all aliases
- Sort them
- Create an array

I suggest:
- Create an array
- Sort by name

Should be more laconically

Reply:
This part is rewritten.